mmv1/templates/terraform/examples/scc_v2_project_notification_config_basic.tf.erb

```diff
@@ -0,0 +1,15 @@
+resource "google_pubsub_topic" "scc_v2_project_notification" {
+  name = "<%= ctx[:vars]['topic_name'] %>"
+}
+
+resource "google_scc_v2_project_notification_config" "<%= ctx[:primary_resource_id] %>" {
+  config_id    = "<%= ctx[:vars]['config_id'] %>"
+  project      = "<%= ctx[:test_env_vars]['project'] %>"
```

Comment:
failing tests are happening due to the missing project_id, can you address the tests in order to include this field?

Reply:
@BBBmau fixed it, please approve the trigger once preliminary checks are done.